server.js

```diff
@@ -1,5 +1,6 @@
 import express from "express";
```

Comment:
It's recommended to use single quotes (') instead of double quotes (") for import statements.